src/ngx_http_lua_worker.c

```diff
@@ -130,4 +130,41 @@ ngx_http_lua_ffi_worker_count(void)
 
     return (int) ccf->worker_processes;
 }
+
+
+int
+ngx_http_lua_ffi_get_process_type(void)
+{
+    int           typ = ngx_process;
+
+    if (typ == NGX_PROCESS_HELPER) {
+#ifdef HAVE_PRIVILEGED_PROCESS_PATCH
+        if (ngx_is_privileged_agent) {
+            typ = 99;
```

Comment:
sorry for this, I'll pay more attentions on those points. 05920eef4b0c2dd2e7cdb67258f3b8b1480ac0fb